utils/scripts/helpers/repo-helper-el8.sh

```diff
@@ -70,14 +70,18 @@ if [ ! -f /etc/fedora-release ]; then
         PT_REPO=powertools
     fi
     dnf install epel-release
-        dnf config-manager --enable $PT_REPO
+        dnf config-manager --enable "$PT_REPO"
     fi
     dnf clean all
 
 daos_base="job/daos-stack/job/"
 artifacts="/artifact/artifacts/el8/"
 save_repos=()
 for repo in $REPOS; do
+    # don't install daos@ repos since we are building daos
+    if [[ $repo = daos@* ]]; then
+        continue
+    fi
```

Comment:
Then why are the daos repos added to REPOS, only to be removed?

Reply:
Because of https://github.com/daos-stack/daos/blob/3c2d2d335b1d92699b23f52f8565917c99c2d808/Jenkinsfile#L700 which gets invoked when a PR wants to use another build's RPMS for it's functional testing but does not skip the non-RPM builds.

Prior to this PR this really was a corner-case as I would suspect that most times people wanting to speed up their runs by using a previous runs's RPMs probably skip all of the building stages entirely.

I experimented in this PR with not even building RPMs if none of the Functional tests were going to run which found this problem.  I have since decided that building RPMs for Jenkins was necessary even if the Functional testing was going to run in GHA.

However none of that changed the fact tht this was a bug that could happen in certain use-cases and seemed worth fixing regardless.